Libs/Core/ctkModelTester.cpp

```diff
@@ -258,7 +258,7 @@ void ctkModelTester::testData(const QModelIndex& index)const
     {
     this->test(index.data(Qt::DisplayRole).isValid(), 
                QString("A valid index can't have invalid data: %1, %2, %3")
-               .arg(index.row()).arg(index.column()).arg(long(index.internalPointer())));
+               .arg(index.row()).arg(index.column()).arg(ptrdiff_t(index.internalPointer())));
```

Comment:
Should it be `std::ptrdiff_t` ? See http://en.cppreference.com/w/cpp/types/ptrdiff_t

Reply:
I'm really no expert in QString argument parsing :-)  According to a brief google search there are other, potentially more standard ways to represent a pointer, i.e. https://stackoverflow.com/questions/8881923/how-to-convert-a-pointer-value-to-qstring

I oriented myself on the existing code, that currently prints the address as a "classic" integer. `ptrdiff_t` would be a signed integer type that is guaranteed to be sized correctly for holding a pointer on all platforms, so it seems fitting. In other words, I intended to make the smallest possible improvement that fixes the issue but keeps all other behavior intact.

However if we assume that the pointer may never be negative (since its not a pointer arithmetic result) then `size_t` would work better. What do you think?

Reply:
> ptrdiff_t would be a signed integer type that is guaranteed to be sized correctly for holding a pointer on all platforms, so it seems fitting.

Agreed. I think the proposed change is good enough. 

Since CI is green, and after checking it also compile and run on windows. I will integrate.

Windows results:
```
    Start 19: ctkModelTesterTest1
1/2 Test #19: ctkModelTesterTest1 ..............   Passed    0.02 sec
    Start 20: ctkModelTesterTest2
2/2 Test #20: ctkModelTesterTest2 ..............   Passed    0.01 sec
```